wolfcrypt/src/port/pic32/pic32mz-crypt.c

```diff
@@ -301,13 +301,12 @@ typedef struct {
     securityAssociation         sa                 __attribute__((aligned (8)));
 } pic32mz_desc;
 
-static pic32mz_desc gLHDesc;
+static pic32mz_desc gLHDesc __attribute__((coherent));
```

Comment:
Is this attribute going to be compiler dependent?

Reply:
Yes it is. This code is only included / supported with the PIC32 and the Microchip XC32 compiler.